doc/modules/mod_global_distrib.md

```diff
@@ -104,53 +104,169 @@ Global distribution modules expose several per-datacenter metrics that can be us
 
 ### Options
 
-* **global_host** (string, required): The XMPP domain that will be shared between datacenters.
-  *Note:* this needs to be one of the domains given in `host` option in `mongooseim.cfg`.
-* **local_host** (string, required): XMPP domain that maps uniquely to the local datacenter; it will be used for inter-center routing.
-  *Note:* this needs to be one of the domains given in `host` option in `mongooseim.cfg`.
-* **message_ttl** (integer, default: `4`): Number of times a message can be rerouted between datacenters.
-* **connections** (list, default: `[]`): Options for connections maintained by the module; see *Connections' options* section.
-* **cache** (list, default: `[]`): Options for caching database lookups; see *Database cache options* section.
-* **bounce** (list | `false`, default: `[]`): Options for message bouncing; if `false`, message bouncing is disabled.
-  See *Message bouncing options* section.
-* **redis** (list, default: `[]`): Options for Redis session storage backend.
-* **hosts_refresh_interval** (integer, default: 3000) - The interval (in milliseconds) telling how often Redis should be asked if new hosts appeared.
+#### `modules.mod_global_distrib.global_host`
+* **Syntax:** string
+* **Default:** none, this option is mandatory
```

Comment:
Is `empty` allowed? If no, please change to `non-empty string`

Reply:
Technically it is. [hosts docs](https://github.com/esl/MongooseIM/blob/doc-toml-modules/doc/advanced-configuration/general.md#hosts) do not specify that and in the code of `mod_global_distrib` empty string would be fine, although it does not make much sense